Mathlib/Order/RelIso/Basic.lean

```diff
@@ -341,6 +349,12 @@ protected theorem isWellOrder : ∀ (_ : r ↪r s) [IsWellOrder β s], IsWellOrd
 
 end RelEmbedding
 
+/-- The induced relation on a subtype is an embedding under the natural inclusion. -/
+@[simps!]
+def Subtype.relEmbedding {X : Type*} (r : X → X → Prop) (p : X → Prop) :
+    (Subtype.val : Subtype p → X) ⁻¹'o r ↪r r :=
+  ⟨Embedding.subtype p, Iff.rfl⟩
```

Comment:
nit:
```suggestion
  ⟨.subtype p, .rfl⟩
```

Reply:
Given this code is moved, I'd prefer to leave it as is. I also think the full `Embedding.subtype` draws attention to a naming inconsistency here, which seems like a good thing.